lib/index.js

```diff
@@ -21,6 +21,49 @@ module.exports = new class {
       "copy-path:copy-full-dirname": (e) => this.copyFullDirname(e),
       "copy-path:copy-line-reference": (e) => this.copyLineReference(e)
     }));
+    if (process.platform === "win32") {
+      this.activateForWin();
+    }
+  }
+
+  activateForWin() {
+    this.subscriptions.add(atom.commands.add("atom-workspace", {
+      "copy-path:copy-project-relative-path-web": (e) => this.copyProjectRelativePathForWeb(e)
+    }));
+    atom.packages.onDidActivatePackage((package) => {
```

Comment:
A bit hacky way but there would be no way to separate menu cson files between platforms.
cf. Package#loadMenu: https://github.com/atom/atom/blob/v1.5.3/src/package.coffee#L247-L252